### PR TITLE
In exports if no getter is found attempt to use setter as getter

### PR DESCRIPTION
**Pull request by caupo-roy**
Also adding a missing return statement to the get method of LocalizedfieldSetter

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
How should a setter be able to do exports? The Setters responsibility is to set a value in a Pimcore object, not to get it

**Comment by caupo-roy** [comment]
Indeed. But the things is that LocalizedfieldSetter, and several other setters, already implement the GetterInterface. This means they are already both setters and getters. My suggestion is simply to make use of this. Think of it as a sane default for the getter.

**Comment by dpfaffenbauer** [comment]
Yep, but they are registered anyway as getter, so I don’t see the point for this

File: src/ImportDefinitionsBundle/Setter/LocalizedfieldSetter.php [hunk 1]
```diff
@@ -39,12 +39,11 @@ public function set(Concrete $object, $value, Mapping $map, $data)
     public function get(Concrete $object, Mapping $map, $data)
     {
         $config = $map->getSetterConfig();
-
         $getter = explode('~', $map->getToColumn());
         $getter = sprintf('get%s', ucfirst($getter[0]));
 
         if (method_exists($object, $getter)) {
-            $object->$getter($config['language']);
+            return $object->$getter($config['language']);
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Setter/LocalizedfieldSetter.php]: this is valid -> everything else is pointless in my opinion

File: src/ImportDefinitionsBundle/Export/Exporter.php [hunk 2]
```diff
@@ -239,7 +247,13 @@ private function getObjectValue(Concrete $object, Mapping $map, $data, Definitio
         $value = null;
 
         if ($map->getGetter()) {
-            $setter = $this->getterRegistry->get($map->getSetter());
+            $getter = $this->getterRegistry->get($map->getGetter());
```
Comment (caupo-roy) [inline src/ImportDefinitionsBundle/Export/Exporter.php]: This one is probably valid as well right? You shouldn't call getSetter on map here

Comment (dpfaffenbauer) [reply src/ImportDefinitionsBundle/Export/Exporter.php]: right

**Comment by dpfaffenbauer** [comment]
I will close these PR, can you create another one for the 2 valid points?

**Pull request closed without merging**